Update registration and login information (#767)
validmind/documentation: site/guide/configuration/register-with-validmind.qmd

Register with our cloud-hosted {{< var validmind.platform >}}, so you can begin working with model documentation or configure {{< var vm.product >}} for your organization.

With an email address or a Google,[^1] GitHub,[^2] or Microsoft[^3] account, you can only gain access to {{< var vm.product >}} by:

> 1. [Accepting an invite](#accept-an-invite)

Reviewer: This shouldn't be a numbered list anymore — please edit this section to explain that you can only gain access by accepting an invite *from your organization*.

::: {.attn}

## Prerequisites

- [x] Your organization has an existing account with {{< var vm.product >}}.
- [x] You have been invited by someone else in your organization to join {{< var vm.product >}}.
- [x] Additionally, an email address that is able to receive messages from {{< var vm.product >}}.[^4]

::: {.callout}

> ValidMind access is invite-only. To use the platform, your organization must already have a ValidMind account, and you must be invited by someone with the Customer Admin role[^5].

Reviewer: ```suggestion
## {{< var vm.product >}} access is invite-only. 

To use the platform, your organization must already have a {{< var vm.product >}} account, and you must be invited by someone with the Customer Admin role[^5].
```

Reviewer: **Please make sure you're following our conventions as outlined in our style guide.** You'll note here that you missed the variables: https://docs.validmind.ai/about/contributing/style-guide/conventions.html#product-names

Register with our cloud-hosted {{< var validmind.platform >}}, so you can begin working with model documentation or configure {{< var vm.product >}} for your organization.

> To access {{< var vm.product >}}, you must be invited by someone in your organization. Once you [receive an invitation](#accepting-an-invite), you can register using your email address or an existing Google,[^1] GitHub,[^2] or Microsoft[^3] account.

Reviewer: ```suggestion
To access {{< var vm.product >}}, an administrator of your organization must invite you to join. Once you receive an invitation, register using your email address or an existing Google,[^1] GitHub,[^2] or Microsoft[^3] account.
```

Reviewer: Again the style guide is your best friend! This time, please refer to the section on inline links: https://docs.validmind.ai/about/contributing/style-guide/conventions.html#inline-links

In this case, no footnote is required — users are already going to see the instructions when they look down two lines.

In general:
- Active voice is preferred to the passive, unless it's super duper awkward. This shifts the focus of the action to the subject (in this case, the "administrator of your organization"), making it clear *who* should perform the action — we are also explicit here as to who that "someone" might be. 
- Use the directly emphatic action word (sometimes known as the "imperative" — we are giving an instruction) as lends clarity and again shifts agency to the user/reader. It says more in less words. :)

::: {.attn}

## Prerequisites

- [x] Your organization has an existing account with {{< var vm.product >}}.
- [x] You have been invited[^5] by someone else in your organization to join {{< var vm.product >}}.
- [x] Additionally, an email address that is able to receive messages from {{< var vm.product >}}.[^4]

::: {.callout}

> {{< var vm.product >}} access is invite-only. To use the platform, your organization must already have a {{< var vm.product >}} account, and you must be invited by someone with the Customer Admin role[^6].

Reviewer: ```suggestion
## {{< var vm.product >}} access is invite-only. 

To use the platform, your organization must already have a {{< var vm.product >}} account, and you must be invited by someone with the Customer Admin role[^6].
```

Reviewer: You forgot to commit this suggestion — please make sure you're applying changes by clicking on the **Commit suggestion** button: **[READ THIS AND FOLLOW THE INSTRUCTIONS](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes)**

It's important that you do this *especially* when collaborating, so everyone gets credit for their work.
